### PR TITLE
Add kubeconfigID override to busola-migrator

### PR DESCRIPTION
**Pull request by moelsayed**
**Description**
Add kubeconfigID override to busola-migrator
Changes proposed in this pull request:

- Add kubeconfigID override to busola-migrator template
- Add kubeconfigID override to busola-migrator code
- Remove old initial kubeconfig URL generator

**Related issue(s)**

### CODE REVIEW

File: resources/busola-migrator/values.yaml [hunk 1]
```diff
@@ -8,14 +8,15 @@ global:
...
     repository: "eu.gcr.io/kyma-project/busola-migrator"
-    tag: "a5b820ae"
+    tag: "PR-11619"
     pullPolicy: IfNotPresent
   env:
     busolaURL: "https://dashboard.kyma.cloud.sap"
     oidcIssuerURL: "https://kyma.accounts.ondemand.com"
     oidcClientId: "12b13a26-d993-4d0c-aa08-5f5852bbdff6"
     oidcScope: "openid email"
     oidcUsePKCE: "true"
```
Comment (tgorgol) [inline resources/busola-migrator/values.yaml:18]: If we no longer use oidc related values in the business logic I'd remove them from configs.

File: resources/busola-migrator/templates/deployment.yaml [hunk 2]
```diff
@@ -59,6 +59,8 @@ spec:
               value: {{ .Values.deployment.env.oidcScope | quote }}
             - name: APP_OIDC_USE_PKCE
               value: {{ .Values.deployment.env.oidcUsePKCE | quote }}
```
Comment (tgorgol) [inline resources/busola-migrator/templates/deployment.yaml:61]: If we no longer use oidc related values in the business logic I'd remove them from configs.

File: components/busola-migrator/internal/config/config.go [hunk 3]
```diff
@@ -19,6 +19,7 @@ type Config struct {
 	TimeoutIdle    time.Duration `envconfig:"default=120s"`
 	BusolaURL      string        `envconfig:"default=https://busola.main.hasselhoff.shoot.canary.k8s-hana.ondemand.com"`
 	StaticFilesDIR string        `envconfig:"optional"`
+	KubeconfigID   string        `envconfig:""`
 
 	OIDC OIDCConfig
```
Comment (tgorgol) [inline components/busola-migrator/internal/config/config.go:24]: If we no longer use oidc related values in the business logic I'd remove them from configs.

**Review by tgorgol — APPROVED** [review]

**Review by tgorgol — APPROVED** [review]

**Review by alexandra-simeonova — APPROVED** [review]

**Pull request merged**